<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Autocompleter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="buttonLayout">
        <item>
         <widget class="QPushButton" name="button1">
          <property name="text">
           <string>F1: Word</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button2">
          <property name="text">
           <string>F2: Word</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button3">
          <property name="text">
           <string>F3: Word</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button4">
          <property name="text">
           <string>F4: Word</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button5">
          <property name="text">
           <string>F5: Word</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTextEdit" name="textArea">
          <property name="font">
           <font>
            <family>Courier New</family>
            <pointsize>12</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="actionNewFile"/>
    <addaction name="actionOpenFile"/>
    <addaction name="actionSaveFile"/>
   </widget>
   <addaction name="menuPlik"/>
  </widget>
  <action name="actionSaveFile">
   <property name="text">
    <string>Zapisz jako...</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="text">
    <string>Otw√≥rz...</string>
   </property>
  </action>
  <action name="actionNewFile">
   <property name="text">
    <string>Nowy</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>textArea</sender>
   <signal>textChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>text_area_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button1</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>button1_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>button2_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>button3_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>button4_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>445</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button5</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>button5_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>570</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNewFile</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>new_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpenFile</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSaveFile</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>save_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
